src/main/java/com/pollapp/controller/PollController.java

```diff
@@ -3,24 +3,34 @@
 import com.pollapp.entity.Category;
 import com.pollapp.entity.Comment;
 import com.pollapp.entity.Poll;
+import com.pollapp.repository.PollRepository;
+
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
+@CrossOrigin
 @RestController
 @RequestMapping("/polls")
 public class PollController {
 
-    @GetMapping("")
+	@Autowired
+	private PollRepository pollRepository;
+	
+    @GetMapping("/ongoing")
     public List<Poll> getPoll() {
-        // TODO
-        return null;
+        return pollRepository.findAll();
+    }
+    
+    @GetMapping("/closed")
+    public List<Poll> getClosedPoll() {
+        return pollRepository.findAll();
     }
 
-    @PostMapping("")
-    public Poll createPoll() {
-        // TODO
-        return null;
+    @PostMapping("/create")
+    public void createPoll(@RequestParam Poll poll) {
```

Comment:
@PostMapping("/") is enough to keep api clean

Reply:
Thanks. I've changed it.

Reply:
:+1: 
 I also updated url in script.